src/components/TransactionFlow/MultiTxReceipt.tsx

```diff
@@ -99,7 +100,17 @@ export default function MultiTxReceipt({
         const txUrl = buildTxUrl(network.blockExplorer, transaction.txHash!);
 
         const assetRate = getAssetRate(asset);
-
+        const recipient = receiverAddress || to!
```

Comment:
```suggestion
        const recipient = receiverAddress ?? to!
```

Reply:
It shouldn't ever get to that point at this point in the flow, because the txs at this point are already created. I think it's factor of having one type for state factories in place of many where it relates to different states of tx creation/broadcast/response. Been a while since I've looked at it though